### PR TITLE
vimPlugins.vim-instant-markdown: init at 2019-09-06

### PR DESCRIPTION
**Pull request by iv-nn**
###### Motivation for this change

Add [vim-instant-markdown](https://github.com/suan/vim-instant-markdown) plugin.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by jonringer — COMMENTED** [review]
this looks like an addition of a node package, and usage of said package in a vim module. The commit history should look like:

```
nodePackages.instant-markdown-d: init at XXXXX
vimPlugins: update
vimPlugins.vim-instant-markdown: init at 2019-09-06
```

**Comment by iv-nn** [comment]
I just rebased to fix conflicts.

**Comment by jonringer** [comment]
I just merged another big PR, do you mind rebasing again

**Comment by iv-nn** [comment]
Done.

File: pkgs/development/node-packages/composition-v13.nix [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 {pkgs ? import <nixpkgs> {
     inherit system;
-  }, system ? builtins.currentSystem, nodejs ? pkgs."nodejs-13_x"}:
+  }, system ? builtins.currentSystem, nodejs ? pkgs."nodejs-8_x"}:
```
Comment (iv-nn) [inline pkgs/development/node-packages/composition-v13.nix]: This is weird and might be related to #70974 or #81275.

**Comment by jonringer** [comment]
@GrahamcOfBorg eval

**Comment by iv-nn** [comment]
I finally got to rebase it, had issue with the node packages `generate.sh` script and then with github rate limiting when running vim plugins `update.py` (until I noticed the new `--proc` option).

**Comment by prusnak** [comment]
Please rework your PR. It now has a merge conflict after PR https://github.com/NixOS/nixpkgs/pull/89184 has been merged

**Comment by cideM** [comment]
@iv-nn It's a hassle with the constant merge conflicts :( but are you still available for seeing this through to the end? Otherwise I'd just pick it up

**Comment by jonringer** [comment]
usually the conflicts aren't too bad with vim plugins, but they are horrendous with node, as it needs to re-make the entire node world.

**Comment by iv-nn** [comment]
I tried to rebase and run `generate.sh` but then I've changes in `node-env.nix`, `utillinux` is renamed to `util-linux` going against  bc49a08. I'm not sure what to do.

**Comment by teto** [comment]
@iv-nn place yourself on top of master, cherry-pick the code that hasn't changed and redo what you've done the first time. I can try to merge it quick if that works fine.

**Comment by teto** [comment]
feel free to reopen when rebasing/updating the PR

**Pull request closed without merging**